app/views/users/new.html.haml

```diff
@@ -1,7 +1,13 @@
+<script type="text/javascript" src="https://js.stripe.com/v2/"></script>
+
+<script type="text/javascript">Stripe.setPublishableKey("#{ENV['STRIPE_PUBLISHABLE_KEY']}");</script>
+
+= javascript_include_tag 'payment'
```

Comment:
Thats what I am saying, you have to plug it to the pipeline. Otherwise it will not be loaded in production.